core.py

```diff
@@ -713,7 +713,7 @@ class QVariantPrinter:
     def __init__(self, val):
         self.val = val
 
-    def children(self):
+    def tmp(self):
```

Comment:
What does this do? The `to_string` change below seems a copy/paste, does it need some other form of transformation?